config/certmanager/certificate-webhook.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  labels:
+    app.kubernetes.io/name: metal-operator
+    app.kubernetes.io/managed-by: kustomize
+  name: serving-cert  # this name should match the one appeared in kustomizeconfig.yaml
```

Comment:
Should this be more similar with the `secretName` field?

Reply:
yes, changed it